### PR TITLE
add palette: Dracula

### PR DESCRIPTION
**Pull request by adtya**
PR adding palette for the Dracula color theme (https://draculatheme.com)

### CODE REVIEW

File: src/palettes/dracula.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "Comment": "#6272a4",
+    "Cyan": "#8be9fd",
+    "Green": "#50fa7b",
+    "Orange": "#ffb86c",
+    "Pink": "#ff79c6",
+    "Purple": "#bd93f9",
+    "Red": "#ff5555",
+    "Yellow": "#f1fa8c"
+ }
+}
```
Comment (doprz) [inline src/palettes/dracula.json:15]: Please try formatting this file to look similar to the other palettes:
- Indent size = 4
- Uppercase hex color

**Comment by doprz** [comment]
Thank you for your contribution to `dipc`!

I've left some comments on your PR before I can merge the PR.

**Review by doprz — CHANGES_REQUESTED** [review]

**Review by doprz — APPROVED** [review]

**Comment by adtya** [comment]
Hi,
I've updated the PR with the suggested changes. can you take a look now?
:)

**Comment by doprz** [comment]
Once again thank you for the PR and for updating it with the suggested changes.
Approved!

**Pull request merged**